meeting-notes/2021-06-15_meeting_notes.md

```diff
@@ -0,0 +1,56 @@
+# OpenJS Foundation Standards Working Group Meeting 2021-06-15
+
+## Links
+
+* **Recording**: https://www.youtube.com/watch?v=jHto2zR-nN4
+* **GitHub Issue**: https://github.com/openjs-foundation/standards/issues/145
+
+## Present
+
+* OpenJS Foundation Cross Project Council
+* OpenJS Foundation Project Maintainers
+* OpenJS Foundation Board of Directors
+
+* Jory Burson (@jorydotcom)
+* Joe Sepi (@joesepi) 
+* Mike Samuel (@mikesamuel) 
+* Brian Kardell (@bkardell) 
+* Jordan Harband (@ljharb) 
+* Mike Champion (@michaelchampion)
+* Tierney Cyren (@bnb) 
+
+## Agenda
+
+### Announcements
+
+*Extracted from **cross-project-council-agenda** labeled issues and pull requests from the **openjs-foundation org** prior to the meeting.
+
+* marketing committee meeting at 3 pm
+* jory finally updated the notes, there are PRs for notes to review - please +1
+* Check out openjs world videos
+
+### openjs-foundation/standards
+
+* Requesting 10 minutes to discuss API translation effort [#144](https://github.com/openjs-foundation/standards/issues/144)
+
+Mike Samuel shared a presentation on supporting many human languages in development environments. Goal is to help ore-English fluent developers navigate english-heavy apis. Issue of factorials in IDE development. Work happening on cross-language Language translation standards. LSP provides the language smarts & map translations. Additive to translation of documentation. Mike requests people who work as devs while pre-fluent to drive goal setting/prioritization; changes to LSP so a language server can connect identifiers to translations; Developer US expertise; A VSCode plugin; registry website; home org to provide structure. Would be good to have a “test project” from openjs but not necessary. OpenJS Collab Space could be helpful. Microsoft is working on this through VSCode. 
```

Comment:
s/ore/pre/

Re "Microsoft is working on this" I was trying to convey that Microsoft is driving most of the LSP stuff so it'd be nice to involve a Microsoftie, ideally someone on the VSCode or adjacent teams.  They're not working on this AFAICT.

Reply:
Edited Typo + for clarification